src/neptune/metadata_containers/run.py

```diff
@@ -455,9 +459,10 @@ def _get_or_create_api_object(self) -> ApiExperiment:
     def _prepare_background_jobs(self) -> BackgroundJobList:
         background_jobs = [PingBackgroundJob()]
 
-        websockets_factory = self._backend.websockets_factory(self._project_api_object.id, self._id)
-        if websockets_factory:
-            background_jobs.append(WebsocketSignalsBackgroundJob(websockets_factory))
+        if self._enable_remote_signals:
+            websockets_factory = self._backend.websockets_factory(self._project_api_object.id, self._id)
+            if websockets_factory:
+                background_jobs.append(WebsocketSignalsBackgroundJob(websockets_factory))
```

Comment:
Is this job also created for Model and ModelVersion?

Reply:
Only for Run

Reply:
FYI, I found another place where WS is used: https://github.com/neptune-ai/neptune-client/blob/fe600779c39b4a41b4935f4c4f69849e6b4abdcb/src/neptune/legacy/internal/execution/execution_context.py#L91, but based on what I see it's legacy part

Reply:
Don't bother.